tests/metrics/network/network-metrics-cpu-consumption.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+#  Measures cpu % consumption using an inter (docker<->docker) 
+#  network bandwidth using iperf2
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Port number where the server will run
+port=5001:5001
+# Image name
+image=gabyct/network
+# Total measurement time (seconds)
+# This is requiered in order to decrease standard deviation
+total_time=16
+# Middle measurement time (seconds)
+# This time is required in at half of the run when
+# server and client are more stable
+middle_time=8
+# Process to track
+process_name="qemu-system-x86_64"
+
+function setup {
+	runtime_docker
+	kill_processes_before_start
+}
+
+# This script will perform all the measurements using a local setup
+
+# Measures cpu % consumption while running bandwidth measurements
+# using iperf2
+
+function cpu_consumption {
+	setup
+	total_cpu=$(mktemp)
+	server_name="iperf-server"
+	client_name="iperf-client"
+	# Find qemu's path
+	complete_path=$(whereis ${process_name} | cut -d ':' -f2)
+	
+	server_command="iperf -p ${port} -s"
+	$DOCKER_EXE run -d --name=${server_name} ${image} bash -c "${server_command}" > /dev/null
+	server_address=$($DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql))
+
+	client_command="iperf -c ${server_address} -t ${total_time}"
+	$DOCKER_EXE run -d --name=${client_name} ${image} bash -c "${client_command}" > /dev/null
+
+	# Measurement after client and server are more stable
+	sleep ${middle_time}
```

Comment:
If we really must have `sleeps`, please add an `echo` call before the `sleep` stating as a minimum:

- why the test is sleeping.
- for how long the test is sleeping for.